partiql-eval/src/eval/expr/eval_wrapper.rs

```diff
@@ -0,0 +1,598 @@
+use crate::error::EvaluationError;
```

Comment:
For readability: maybe we can move `eval_wrapper` to a sub-directory or the the other way around; `exprs` to a sub-directory.